static/app/views/issueDetails/useProjectLatestSemverRelease.tsx

```diff
@@ -0,0 +1,31 @@
+import {ReleaseStatus, type Release} from 'sentry/types/release';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export default function useProjectLatestSemverRelease({enabled}: {enabled: boolean}) {
+  const organization = useOrganization();
+  const location = useLocation();
+
+  const {data, isError, isPending} = useApiQuery<Release[]>(
+    [
+      `/organizations/${organization.slug}/releases/`,
+      {
+        query: {
+          per_page: 1,
+          sort: 'semver',
+          project: location.query.project,
+          environment: location.query.environment,
+          status: ReleaseStatus.ACTIVE,
+        },
+      },
+    ],
```

Comment:
This won't work for the `sentry` or `javascript` projects both of which have years old semver releases. That would signal to your code that you need to show the semver button (I assume -- I don't actually understand the code) even though we don't use semver.